docs/tutorials/transaction-tutorial.md

```diff
@@ -7,32 +7,37 @@ description: Learn how to perform database operations
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-## Transaction Functions
-
-To perform operations on your application database in DBOS, you use _transaction functions_.
-As their name implies, these functions execute as [database transactions](https://en.wikipedia.org/wiki/Database_transaction).
+Use _transaction functions_ to read and write from your database. A transaction function may contian multiple queries, punctuated by local logic, and executes as a single [database transaction](https://en.wikipedia.org/wiki/Database_transaction). 
 
 Transaction functions must be annotated with the [`@Transaction`](../api-reference/decorators#transaction) decorator and must have a [`TransactionContext`](../api-reference/contexts#transactioncontextt) as their first argument.
 As with other DBOS functions, inputs and outputs must be serializable to JSON.
 
-The [`TransactionContext`](../api-reference/contexts#transactioncontextt) provides a `.client` field you can use to transactionally interact with the database, so you don't need to worry about managing database connections.
+The [`TransactionContext`](../api-reference/contexts#transactioncontextt) provides a `.client` field you can use to interact with the database, so you don't need to worry about managing connections.
 DBOS supports [Knex.js](./using-knex.md), [Drizzle](./using-drizzle.md), [TypeORM](./using-typeorm.md), and [Prisma](./using-prisma.md) clients as well as raw SQL.
 You can configure which client to use in your [`dbos-config.yaml`](../api-reference/configuration.md) file.
-Knex is used by default.
```

Comment:
To be honest, I think we recommend using [Drizzle `sql` templates](https://orm.drizzle.team/docs/sql) for raw SQL...